client/state/current-user/constants.js

```diff
@@ -1 +1,2 @@
 export const DOMAINS_WITH_PLANS_ONLY = 'calypso_domains_with_plans_only';
+export const PLAN_UPSELL_FOR_FREE_USERS = 'calypso_plan_upsell_for_non_dwpo_free_users';
```

Comment:
Could we rename this flag to something more accurate? The general plan is to sell domains to free users, not upsell them plans.

Reply:
Done!